app/services/notification/NotificationScheduler.scala

```diff
@@ -0,0 +1,39 @@
+package services.notification
+
+import config.AMIableConfig
+import models.NotifySchedule
+import org.quartz.CronScheduleBuilder.cronSchedule
+import org.quartz.JobBuilder.newJob
+import org.quartz.TriggerBuilder.newTrigger
+import org.quartz.{JobKey, TriggerKey}
+import org.quartz.impl.StdSchedulerFactory
+import play.api.Logger
+
+import scala.concurrent.ExecutionContext
+
+class NotificationScheduler {
```

Comment:
Is this *meaningfully* different to the `Notifications` class - what does the Notifications class (/service) *do* if not schedule notifications? Is there a reason why they need to be separate?

It might be simpler as one class. Also, combining them into a single Guice-y class would go some way towards solving the multiple instance thing I mention below.

Looking for a conversation here (as ever!), I'm certainly not just saying "do what I think" :-)